src/sheet_metal_conus.py

```diff
@@ -57,29 +84,130 @@ def __init__(self):
             type = 'inkbool', dest = 'verbose', default = False,
             help = 'Enable verbose output of calculated parameters. Used for debugging or is someone needs the calculated values.')
 
-
     def getUnittouu(self, param):
         " compatibility between inkscape 0.48 and 0.91 "
         try:
             return inkex.unittouu(param)
         except AttributeError:
             return self.unittouu(param)
 
-    def unsignedLong(self, signedLongString):
-        " interpret the signed long as unsigned "
-        longColor = long(signedLongString)
-        if longColor < 0:
-            longColor = longColor & 0xFFFFFFFF
-        return longColor
-
-    #A*256^0 + B*256^1 + G*256^2 + R*256^3
-    def getColorString(self, longColor):
-        " convert the long into a #RRGGBB color value "
-        longColor = self.unsignedLong(longColor)
+    def getColorString(self, longColor, verbose=False):
+        """ Convert the long into a #RRGGBB color value
+            - verbose=true pops up value for us in defaults
+            conversion back is A + B*256^1 + G*256^2 + R*256^3
+        """
+        if verbose: inkex.debug("%s ="%(longColor))
+        longColor = long(longColor)
+        if longColor <0: longColor = long(longColor) & 0xFFFFFFFF
         hexColor = hex(longColor)[2:-3]
-        hexColor = hexColor.rjust(6, '0')
-        return '#' + hexColor.upper()
+        hexColor = '#' + hexColor.rjust(6, '0').upper()
+        if verbose: inkex.debug("  %s for color default value"%(hexColor))
+        return hexColor
+
+    # Marker arrows
+    def makeMarkerstyle(self, name, rotate):
```

Comment:
The following functions deal with arrows.
First one defines new marker style which is DIN arrow